templates/warp.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>warp</Name>
+  <Repository>caomingjun/warp</Repository>
+  <Registry>https://hub.docker.com/r/caomingjun/warp/</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://hub.docker.com/r/caomingjun/warp/</Support>
+  <Project>https://github.com/cmj2002/warp-docker/</Project>
+  <Overview>&#xD;
+  Cloudflare WARP is a free VPN service provided by Cloudflare. As most service providers consider its exit IP as a reputable residential broadband IP, many people use it for accessing websites that have strict risk control policies, especially when their server’s IP address is not clean.
+  &#xD;
+  </Overview>
+  <ExtraParams>--sysctl net.ipv6.conf.all.disable_ipv6=0 --sysctl net.ipv4.conf.all.src_valid_mark=1 --cap-add NET_ADMIN</ExtraParams>
+  <Config Name="WARP Sleep Cycle" Target="WARP_SLEEP" Default="2" Mode="" Description="" Type="Variable" Display="always" Required="true" Mask="false">2</Config>
+  <Config Name="Appdata Directory" Target="/data" Default="/mnt/user/appdata/warp/" Mode="rw" Description="" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/warp/</Config>
```

Comment:
```suggestion
  <Config Name="Appdata Directory" Target="/var/lib/cloudflare-warp" Default="/mnt/user/appdata/warp/" Mode="rw" Description="" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/warp/</Config>
```

Container map is incorrect: https://github.com/cmj2002/warp-docker/?tab=readme-ov-file#start-the-container